modules/Bio/EnsEMBL/Compara/PipeConfig/nextflow_test.nf

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env nextflow
+
+// Commandline parameters here
+params.dir = '/hps/nobackup/flicek/ensembl/compara/shared/reference_fasta_symlinks/'
+params.orthofinder_exe = '/hps/software/users/ensembl/ensw/C8-MAR21-sandybridge/linuxbrew/bin/orthofinder'
+params.destination = '/hps/nobackup/flicek/ensembl/compara/cristig/reference_collections'
+
+// Assign parameters to variables for later use
+orthofinder_exe = params.orthofinder_exe
+base_dir = file(params.dir)
+dest_dir = file(params.destination)
+
+// A little groovy scripting to collect just the file names from the directories for ease of use,
+// as far as I can tell the file handling in nextflow is much more file rather than directory friendly
+def dir_list = [];
+base_dir.eachFile { item ->
+    if( item.isDirectory() ) {
+        if( !item.getName().equals('references') ) {
```

Comment:
```suggestion
    if ( item.isDirectory() ) {
        if ( !item.getName().equals('references') ) {
```
(not sure it can be done, but if you can, pretty please?? :pleading_face:)

Reply:
I think it can be done, but the groovy style preference is without the spaces :shrug: 
My preference is with the spaces though...